src/ccnl-utils/src/ccn-lite-mkI.c

```diff
@@ -109,20 +93,16 @@ main(int argc, char *argv[])
         default:
 Usage:
             fprintf(stderr, "usage: %s [options] URI\n"
-            "  -a LEN     miN additional components\n"
-            "  -c SCOPE\n"
-
             "  -d DIGEST  content digest (sets -x to 0)\n"
             "  -e NONCE   random 4 bytes\n"
             "  -l         URI is a Lambda expression\n"
             "  -n CHUNKNUM positive integer for chunk interest\n"
             "  -o FNAME   output file (instead of stdout)\n"
             "  -p DIGEST  publisher fingerprint\n"
-            "  -s SUITE   (ccnb, ccnx2015, ndn2013)\n"
 #ifdef USE_LOGGING
-            "  -v DEBUG_LEVEL (fatal, error, warning, info, debug, verbose, trace)\n"
+            "  -v DEBUG_LEVEL (fatal, error, warning, info, debug, verbose, trace)\n",
 #endif
-            "  -x LEN     maX additional components\n",
+            "  -s SUITE   (ccnb, ccnx2015, ndn2013)\n",
             argv[0]);
```

Comment:
Here is sth wrong with the brackets

Reply:
Didn't change it, but simply yanked the line in vim. Do you want to have it like this
```C
"  -s SUITE (ccnb, ccnx2015, ndn2013)\n",
```